src/components/NotesPage/NotesPageTopBar.js

```diff
@@ -16,34 +18,60 @@ import { LogoutButtonContainer } from 'components/LogoutButton/LogoutButtonConta
 
 import * as Selectors from 'components/Uploads/UploadsSelectors';
 
+import { colors } from 'colors';
+
 export const NotesPageTopBar = () => {
   const hasUploads = useSelector(Selectors.hasUploads);
   // const hasActiveUploads = useSelector(Selectors.hasActiveUploads);
 
+  const [state, setState] = React.useState(0);
+
+  const handleClick = () => {
```

Comment:
Let's make it clear which click are we handling, by renaming it to something `handleMenuIconClick`.

Reply:
Renamed.